Source/WebKit/UIProcess/ios/WKTouchEventsGestureRecognizer.mm

```diff
@@ -339,7 +339,7 @@ - (void)_recordTouches:(NSSet<UITouch *> *)touches type:(WebKit::WKTouchEventTyp
             touchPoint.azimuthAngle = [touch azimuthAngleInView:self.view.window];
         } else {
             touchPoint.touchType = WebKit::WKTouchPointType::Direct;
-            touchPoint.altitudeAngle = 0;
+            touchPoint.altitudeAngle = piOverTwoDouble;
```

Comment:
This default value is non-obvious. Maybe make a remark in a comment that this is prescribed by the Pointer Events spec? Or even link to the spec text if you feel that's appropriate https://w3c.github.io/pointerevents/#dom-pointerevent-altitudeangle:~:text=angle%2C%20the%20value-,MUST%20be%20%CF%80/2,-.